common-libs/internal-api-clients/src/helpers/client.ts

```diff
@@ -0,0 +1,149 @@
+import keyBy from 'lodash.keyby'
+import mapValues from 'lodash.mapvalues'
+import FetchType from 'node-fetch'
```

Comment:
Can we do this?
```
import type FetchType from 'node-fetch'
```

Because I suppose this is still an import in the compiled JS code?

Reply:
Yes, we should be able to do this.

Reply:
Fixed